pkg/zson/raw.go

```diff
@@ -110,9 +111,10 @@ func NewRawAndTsFromJSON(d *Descriptor, tsCol int, data []byte) (Raw, nano.Ts, e
 }
 
 func NewRawAndTsFromZeekTSV(d *Descriptor, tsCol int, path []byte, data []byte) (Raw, nano.Ts, error) {
-	data = data[:len(data)-1]     // Remove terminal newline.
 	vals := make([][]byte, 0, 32) // Fixed length for stack allocation.
-	vals = append(vals, path)
+	if len(path) > 0 {
+		vals = append(vals, path)
+	}
```

Comment:
I'd like to avoid this if we can since it's on the critical path for boom ingest. I think we can define it away by saying that Zeek data always has a path. I think that's true in practice but will confirm it.

Reply:
The change here is adding the check on len(), is it that branch that you want to avoid?
I'm skeptical that it has a measurable impact on ingest rate but I can take this (and the related test case) out if you feel strongly.

Reply:
That's the branch I'd like to avoid. It has a small but measurable effect. My tune would be different If Zeek logs without a path were a thing, but my reading of https://github.com/zeek/zeek/blob/master/src/logging/writers/ascii/Ascii.cc is that they're not.